icicle/appUtils/ntt/kernel_ntt.cu

```diff
@@ -116,7 +123,7 @@ namespace ntt {
     int tid = blockDim.x * blockIdx.x + threadIdx.x;
     if (tid >= n_elements * batch_size) return;
     int64_t scalar_id = tid % n_elements;
-    if (rev_type != eRevType::None) scalar_id = generalized_rev(tid, logn, dit, rev_type);
+    if (rev_type != eRevType::None) scalar_id = generalized_rev(tid, logn, dit, false, rev_type);
```

Comment:
shouldn't we be passing fast_tw as a parameter here?

Reply:
it's the kernel that handles cosets but it doesn't work with the fast-twiddles. That's why I removed the parameter and pass false. Otherwise it would calculate incorrectly.